blender/LilySurfaceScrapper/CyclesMaterialData.py

```diff
@@ -65,10 +71,23 @@ def createMaterial(self):
 
             if hasattr(texture_node, "color_space"):
                 texture_node.color_space = "COLOR" if map_name == "baseColor" else "NONE"
+            elif map_name == "glossiness":
+                invert_node = nodes.new(type="ShaderNodeInvert")
+                links.new(texture_node.outputs["Color"], invert_node.inputs["Color"])
+                links.new(invert_node.outputs["Color"], current_principled.inputs["Roughness"])
+                current_principled
+            elif map_name == "height":
+                displacement_node = nodes.new(type="ShaderNodeDisplacement")
+                links.new(texture_node.outputs["Color"], displacement_node.inputs["Height"])
+                links.new(displacement_node.outputs["Displacement"], mat_output.inputs[2])
+                if normal_node is not None:
+                    links.new(normal_node.outputs["Normal"], displacement_node.inputs["Normal"])
             elif map_name == "normal":
                 normal_node = nodes.new(type="ShaderNodeNormalMap")
                 links.new(texture_node.outputs["Color"], normal_node.inputs["Color"])
                 links.new(normal_node.outputs["Normal"], current_principled.inputs["Normal"])
+                if displacement_node is not None:
+                    links.new(normal_node.outputs["Normal"], displacement_node.inputs["Normal"])
```

Comment:
To prevent code duplication, it might be better to test for `normal_node is not None and displacement_node is not None` afterwards, after the end of the loop, to add the link.